parm/gfs/coefb_atms_n20.yaml

```diff
@@ -0,0 +1,42 @@
+diagnostics:
+
+# Data read
+# ---------
+datasets:
+  - name: bcoef
+    satellite: atms
+    sensor: n20
+    type: RadDataSpace
+    filename: /path/to/diag_atms_n20_2021032500.nc
+    channels: &chans [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22]
+    outfile: /path/to/outfile.nc
+    variables to process:
+    - function: calculate_omf
+      groups: ['hofx0', 'hofx1']
```

Comment:
Asking for my own edification, knowing next to nothing about JEDI.  Are the 0 and 1 values roughly analogous to the ges and analysis steps in GSI?

Reply:
yes